README.md

```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+Nikki Schuldt https://github.com/nikkischuldt
```

Comment:
Could you please add a `-` so this becomes a list item?

Also, please make your name into a link that goes to that URL. The URL should not be visible.